mhkit/wave/io.py

```diff
@@ -112,30 +112,32 @@ def read_NDBC_file(file_name, missing_values=['MM',9999,999,99]):
 
 
 def ndbc_available_data(parameter='swden',
-                        number=None, 
+                        buoy_number=None, 
                         proxy=None):  
     '''
-    For a given parameter this will return a DataFrame of years, and 
-    station IDs and file names              
+    For a given parameter this will return a DataFrame of years, 
+    station IDs and file names that contain that parameter data.              
     
     Parameters
     ----------
-    parameter: string
-        'swden'	:	'Raw Spectral Wave Current Year Historical Data'
-    number: string
+    parameter: string (optional)
```

Comment:
As it currently functions this appears optional but this parameter is used in the request address. I would prefer to not list this as optional for this reason as well as there are currently no other data options available.